exercises/bracket-push/src/lib.rs

```diff
@@ -1 +1,13 @@
+pub struct Brackets;
 
+impl From<&'static str> for Brackets {
```

Comment:
I bet this would work, and more generally, without the `'static` lifetime bound.

Reply:
It does not actually. When I try to compile the stub i get the following output:
```rust
error[E0106]: missing lifetime specifier
 --> src/lib.rs:3:11
  |
3 | impl From<&str> for Brackets {
  |           ^ expected lifetime parameter

error: aborting due to previous error

```

Reply:
Right! Sorry about that. Well, let's at least make it a generic lifetime bound, so that someone could generate an arbitrary non-static `&str` and the `From` trait would still work:

```rust
impl<'a> From<&'a str> for Brackets { ... }
```